server/container/guice/cassandra-rabbitmq-guice/src/test/java/org/apache/james/CassandraRabbitMQJmapTestRule.java

```diff
@@ -19,7 +19,9 @@
 
```

Comment:
Given my previous comments, I of course disagree with the purpose of this commit.